pkgs/applications/office/paperwork/paperwork-shell.nix

```diff
@@ -39,6 +40,7 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    shared-mime-info
```

Comment:
Is this needed as part of the update or did you add it simply because it might be useful?

Reply:
the test fail without it